infoschema/tables.go

```diff
@@ -988,6 +991,17 @@ var tableMetricSummaryByLabelCols = []columnInfo{
 	{name: "COMMENT", tp: mysql.TypeVarchar, size: 256},
 }
 
+var tableStorageStatsCols = []columnInfo{
+	{name: "TABLE_SCHEMA", tp: mysql.TypeVarchar, size: 64},
```

Comment:
I think `database` is more user friendly.

Reply:
`TABLE_SCHEMA` is ok here , just keep consistent with another system table.